waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/service/webauthn/WebauthnService.kt

```diff
@@ -9,9 +9,9 @@ import com.webauthn4j.data.client.Origin
 import com.webauthn4j.data.client.challenge.Challenge
 import com.webauthn4j.data.client.challenge.DefaultChallenge
 import com.webauthn4j.server.ServerProperty
-import com.webauthn4j.validator.exception.ValidationException
-
+import com.webauthn4j.validator.exception.ValidationException*/
 
+/* Complete this implementation:
```

Comment:
maybe it is good to have an issue on github, so it doesn't get lost or forgotten?